samples/Network/enforce-nsg-on-all-subnets/azurepolicy.json

```diff
@@ -0,0 +1,44 @@
+{
+    "properties": {
+        "displayName": "Enforce a NSG on all subnets",
+        "description": "Denies the creation of subnets that don't contain a NSG",
+        "mode": "All",
+        "policyRule": {
+            "if": {
+                "anyOf": [
+                    {
+                        "allOf": [
+                            {
+                                "field": "type",
+                                "equals": "Microsoft.Network/virtualNetworks"
+                            },
+                            {
+                                "not": {
```

Comment:
Could you remove "not" and change condition to "exists": "false"

Reply:
Done